docs-client/tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
+  "compileOnSave": false,
+  "compilerOptions": {
+    "allowSyntheticDefaultImports": true,
+    "declaration": false,
+    "esModuleInterop": true,
+    "experimentalDecorators": true,
+    "forceConsistentCasingInFileNames": true,
+    "jsx": "preserve",
+    "lib": ["esnext", "es2017", "es2016", "es2015", "dom"],
+    "module": "esnext",
+    "moduleResolution": "node",
+    "newLine": "LF",
```

Comment:
Question: Does this determine the line separators of compilation output or the `.tsx` files?

Reply:
The compilation output, but since we only pipe though webpack anyways this doesn't matter so removed.

Reply:
Thanks!